wordpress/resources/app.yaml

```diff
@@ -0,0 +1,152 @@
+# for the cluster.
+installer:
+
+  # if 'flavors' section is present, the installer will ask the end user what
+  # type of Kubernetes cluster to create. We are offering either a small
+  # single node, 3 node or 5 node configuration.
+  flavors:
+    prompt: "What size install do you want (small, medium, or large)?"
+    # The default flavor
+    default: small
+    # List of flavors:
+    items:
+    - name: "small"
+      description: "Single node install with Master, MySQL DB and Wordpress PHP"
+      nodes:
+      - profile: general
+        count: 1
+    - name: "medium"
+      description: "Three node install with one Master, one node labeled for DB  and one labeled for front-end usage "
+      nodes:
+      - profile: master
+        count: 1
+      - profile: front
+        count: 1
+      - profile: db
+        count: 1
+#HA Master nodes (3) configuration with a 2 fronts and a db
+    - name: "large"
+      description: "High Availability Master Node configuration with 2 Wordpress PHP nodes and 1 My"
+      nodes:
+      - profile: front
+        count: 2
+      - profile: db
+        count: 1
+      - profile: master
+        count: 3
+
+# This section allows to define what flavors of servers are required for
+# this cluster. This example uses a single flavor called "node" but you
+# can have flavors like "database" or "gpu", etc.
+#
+#
+nodeProfiles:
+  - name: general
+    description: "full node"
+    labels:
+      front: "true"
+      db: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+  - name: master
+    description: "master node"
+    labels:
+      node-role.kubernetes.io/master: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+  - name: front
+    description: "Web front end node"
+    labels:
+      front: "true"
+      node-role.kubernetes.io/node: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+  - name: db
+    description: "Database node"
+    labels:
+      db: "true"
+      node-role.kubernetes.io/node: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+
+
+
+
+# This section allows you to specify Kubernetes jobs that will be executed
+# inside the cluster when certain cluster lifecycle events happen
+hooks:
+  install:
+    job: file://install.yaml
+#As new versions of wordpress are realized this upgrade hook will make s
+  update:
+    job: file://upgrade.yaml
+
+  # in this example the hooks below do not do anything. we're listing them here
+  # to demostrate the possibility of customizing cluster life cycle.
```

Comment:
demo**n**strate

Reply:
hooks removed per Roman's comments so this removed too.